includes/wp-admin/class-settings.php

```diff
@@ -191,6 +191,16 @@ public static function register_settings() {
 			)
 		);
 
+		\register_setting(
+			'activitypub_advanced',
+			'activitypub_content_negotiation',
+			array(
+				'type'        => 'boolean',
+				'description' => \__( 'Enable content negotiation.', 'activitypub' ),
```

Comment:
```suggestion
				'description' => 'Enable content negotiation.',
```
I still don't know if this is user-facing anywhere. Let's keep it untranslated for now?